tests/_fixtures/marathon-pods/groups.js

```diff
@@ -1,5 +1,236 @@
+const createFrameworks = () =>
```

Comment:
just to be sure: there's nothing in here that shouldn't be public, right?

Reply:
I don’t think so but I will all unnecessary information to thin out this change.